config/custom-environment-variables.json

```diff
@@ -56,48 +56,43 @@
   },
   "provider": {
     "source": {
-      "type": "PROVIDER_FROM",
-      "accessKeyId": "S3_SOURCE_ACCESS_KEY_ID",
-      "secretAccessKey": "S3_SOURCE_SECRET_ACCESS_KEY",
-      "endpointUrl": "S3_SOURCE_END_POINT_URL",
-      "bucket": "S3_SOURCE_BUCKET",
+      "kind": "PROVIDER_FROM",
+      "credentials": {
+        "accessKeyId": "S3_SOURCE_ACCESS_KEY_ID",
+        "secretAccessKey": "S3_SOURCE_SECRET_ACCESS_KEY"
+      },
+      "endpoint": "S3_SOURCE_END_POINT_URL",
+      "bucketName": "S3_SOURCE_BUCKET",
       "region": "S3_SOURCE_REGION",
       "forcePathStyle": {
         "__name": "S3_SOURCE_FORCE_PATH_STYLE",
         "__format": "boolean"
       },
-      "sslEnabled": {
-        "__name": "S3_SOURCE_SSL_ENABLED",
-        "__format": "boolean"
-      },
       "maxAttempts": {
         "__name": "S3_SOURCE_MAX_ATTEMPTS",
         "__format": "number"
       },
-      "sigVersion": "S3_SOURCE_SIG_VERSION",
+      "storageClass": "S3_SOURCE_STORAGE_CLASS",
```

Comment:
There is no need for storage class in source because it is relevant only for uplodad requests (put command)

Reply:
as discussed, it can be so it should have